tools/tests/test_representations.py

```diff
@@ -687,6 +738,26 @@ def test_has_latest_dataset(self, mock_static_catalog):
         under_test = instance.has_latest_dataset()
         self.assertFalse(under_test)
 
+    @patch("tools.representations.GtfsRealtimeSource.static_catalog")
+    def test_has_feature(self, mock_static_catalog):
+        test_feature = self.test_feature
+        test_another_feature = ["some_other_feature"]
```

Comment:
Here `has_feature()` take an enum value, not an array.
```suggestion
        test_another_feature = "some_other_feature"
```